modules/openapi-generator/src/main/resources/php/ObjectSerializer.mustache

```diff
@@ -159,8 +165,9 @@ class ObjectSerializer
      */
     public static function toHeaderValue($value)
     {
-        if (method_exists($value, 'toHeaderValue')) {
-            return $value->toHeaderValue();
+        $callable = [$value, 'toHeaderValue'];
```

Comment:
[Using `is_callable` is safer](https://3v4l.org/9NiSX) than [using `method_exists`](https://3v4l.org/ePZkt), while also being more correct.